src/_data/catalog/widgets.json

```diff
@@ -1514,6 +1756,17 @@
     "link": "https://api.flutter.dev/flutter/material/Slider-class.html",
     "image": "<img alt='' src='/assets/images/docs/widget-catalog/material-slider.png'>"
   },
+  {
+    "name": "Slider",
+    "description": "Sliders allow users to make selections from a range of values.",
```

Comment:
**issues:**
- Make consistent with other descriptions.
- Remove the circular definition.
```suggestion
    "description": "Form control that enables selecting a range of values.",
```